src/components/Installer/index.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import styles from "./styles.module.css";
+
+export default function CondaInstallerSelector() {
+  const [os, setOs] = useState("");
+  const [arch, setArch] = useState("");
+  const [installerType, setInstallerType] = useState("");
+
+  const osArchitectures = {
+    Linux: ["x64", "aarch64", "ppc64le"],
+    macOS: ["x64", "ARM64"],
+    Windows: ["x64", "ARM64 (Beta)"],
+  };
+
+  const installerData = {
+    Anaconda: {
+      name: "Anaconda",
+      url: "https://docs.anaconda.com/free/anaconda/install/index.html",
```

Comment:
```suggestion
      url: "https://docs.anaconda.com/anaconda/install/index.html",
```
The `free` directory was removed this year.

Reply:
I also don't know if this matters, but this doc URL (and the Miniconda one below) could also just be "https://docs.anaconda.com/anaconda/install/" and "https://docs.anaconda.com/miniconda/" as the docs use a `dirhtml` build. Either formatting will go to the same place, though.